src/locales/es/translation.json

```diff
@@ -16,7 +16,7 @@
         "linkLabels": ["Enlace Simple tres", "Enlace Simple cuatro"]
       }
     ],
-    "parentLinkLabels": ["Parent link"]
+    "parentLinkLabels": ["Página de Inicio", "Documentación Técnica"]
```

Comment:
Let's double check that these are the correct phrases

Reply:
Verified with Oscar.